Editor/Window/ManagedEC2Page.cs

```diff
@@ -151,12 +152,20 @@ private void UpdateGUI()
             }
             else if (stackStatus.IsStackStatusFailed())
             {
-                _statusIndicator.Set(State.Failed, textProvider.Get(Strings.ManagedEC2DeployStatusFailed));
+                var errorText = textProvider.Get(Strings.ManagedEC2DeployStatusFailed);
+                _statusIndicator.Set(State.Failed, errorText);
+                _deployStatusBox.Show(StatusBox.StatusBoxType.Error,  errorText, null, string.Format(Urls.AwsCloudFormationTemplate, _stateManager.Region, _deploymentSettings.CurrentStackInfo.StackId), Strings.LabelDeploymentCloudFormationConsole);
```

Comment:
The text displayed in the status box should be longer than "Failed" (note that the status indicator should remain the same).

I will confirm with UX on the exact wording to update later, but in the meantime please use the following for the error message: 
```
Something went wrong with the stack. Refer to the CloudFormation console for details. Delete and re-create the deployment once the problem is resolved.
```

Reply:
Updated